GREEDY/App.config

```diff
@@ -8,5 +8,6 @@
         <add key="TessDataPath" value="../../tessdata"/>
         <add key="FilterImageFiles" value="Image Files(*.PNG; *.JPG; *.GIF)| *.PNG; *.JPG; *.GIF"/>
         <add key="SaveFilePath" value="C:\"/>
+        <add key="CategoriesDataPath" value="../../Resources/categories.txt"/>
```

Comment:
need to think about better storing possibilities. Everything will look better in DB later, but now we have to think about productive comparing (categorizing). maybe implement more than one possible way to get category and then to have a final decision witch logical way will be better